packages/@ember/-internals/glimmer/lib/helpers/each-in.ts

```diff
@@ -173,17 +173,5 @@ class EachInReference implements PathReference {
     }
 
     return new EachInWrapper(iterable);
-  }
-
-  get(key: string): PathReference {
-    return this.inner.get(key);
-  }
-}
-
-export class EachInWrapper {
-  constructor(public inner: unknown) {}
-}
-
-export default function(args: VMArguments) {
-  return new EachInReference(args.positional.at(0));
+  });
```

Comment:
Can we give this a label?

Reply:
This helper will never be directly referenced by a user, since it is added via a transform. The only way to reference a helper directly, FWIW, is to do:

```hbs
{{#each (helper) as |helperResult|}}
  {{helperResult.foo}}
{{/each}}
```

So my rule of thumb is, only add labels to helpers that users can actually create themselves and do this with.

Reply:
Hmm, I don't understand. Isn't this talking about `{{#each-in` which is public API?